Makefile

```diff
@@ -25,6 +25,7 @@ all: $(OUTPUT_DIR)/libs $(if $(ANDROID),,$(LUAJIT)) \
 		$(if $(USE_LJ_WPACLIENT),$(LJ_WPACLIENT),) \
 		$(TURBO_FFI_WRAP_LIB) \
 		$(LUA_SPORE_ROCK) \
+		$(LUA_HTMLPARSER_ROCK) \
```

Comment:
Please move it up one line. It's not alphabetical, partially on purpose due to dependency issues, but as a rule of thumb make things alphabetical. It's easier to read. ^_^